java/integTest/src/test/java/glide/cluster/CommandTests.java

```diff
@@ -817,4 +817,23 @@ public void functionLoad(boolean withRoute) {
         assertEquals(libName, promise2.get());
         // TODO test with FCALL
     }
+
+    @Test
+    @SneakyThrows
+    public void randomKey() {
+        String key1 = "{key}" + UUID.randomUUID();
+        String key2 = "{key}" + UUID.randomUUID();
+
+        assertEquals(OK, clusterClient.set(key1, "a").get());
+        assertEquals(OK, clusterClient.set(key2, "b").get());
+
+        String randomKey = clusterClient.randomKey().get();
+        assertEquals(1L, clusterClient.exists(new String[] {randomKey}).get());
+
+        // no keys in database
+        assertEquals(OK, clusterClient.flushall().get());
+        ExecutionException executionException =
+                assertThrows(ExecutionException.class, () -> clusterClient.randomKey().get());
```

Comment:
According to docs, shouldn't the return value be null?

Reply:
it should be, however, when I do an assertNull, the value the program gives is "no value found- ResponseError"

Reply:
Oh there is a bug in https://github.com/Bit-Quill/glide-for-redis/blob/f2c3d49e5961c890b13aa43e4b520bfb903173a6/glide-core/src/client/standalone_client.rs#L257-L270
You need to fix that in scope of this PR

Reply:
I think we need to raise a redis-rs fix for randomKey that should return a null if there are no keys available:
https://github.com/amazon-contributing/redis-rs/blob/main/redis/src/cluster_routing.rs#L321-L322

Change to:
```
            // for randomkey, see: 
            // This isn't based on response_tips, but on the discussion here - https://github.com/redis/redis/issues/12410
            b"FUNCTION KILL" | b"SCRIPT KILL" | b"RANDOMKEY" => Some(OneSucceeded),
```

Reply:
For now, please add a `// TODO` comment that says we need to update redis-rs to return a `OneSucceeded`

Reply:
see: https://github.com/amazon-contributing/redis-rs/pull/149
Feel free to add this link as a TODO comment